docs/source/prereqs.rst

```diff
@@ -83,7 +83,7 @@ Go Programming Language
 Hyperledger Fabric uses the Go Programming Language for many of its
 components.
 
-  - `Go <https://golang.org/dl/>`__ version 1.13.x is required.
+  - `Go <https://golang.org/dl/>`__ version 1.14.x is required.
```

Comment:
This is untrue for app development and the reason why I did not make the update.

Reply:
Agree that the prereq doc overall is a mess and conflates many things. I've opened https://jira.hyperledger.org/browse/FAB-17701 to clean it up.
For now let's just go with the precedent of mentioning the Go version that Fabric uses in the associated release. Although this is not required for chaincode apps, I don't think we could go wrong with having chaincode developers use the same Go version as Fabric itself.